config/end-to-end.ini

```diff
@@ -11,22 +11,24 @@ case=extract_and_output
 extract_options=--no-vss
 output_file=extract_and_output1.log
```

Comment:
Please change this to match the reference_output_file name

Reply:
No, since that would cause the `extract_and_output3` test to overwrite this file

`test_data/end_to_end/extract_and_output.log` is used for both the `extract_and_output1` (psort) and `extract_and_output3` (psteal) tests. Hence the description.